src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7054,6 +7054,7 @@ public sealed partial class StringBuilder : System.Runtime.Serialization.ISerial
         public System.Text.StringBuilder Append(uint value) { throw null; }
         [System.CLSCompliantAttribute(false)]
         public System.Text.StringBuilder Append(ulong value) { throw null; }
+        public System.Text.StringBuilder Append(ReadOnlySpan<char> value) { throw null; }
```

Comment:
Nit: generally we use the full namespace, so System.ReadOnlySpan rather than just ReadOnlySpan.  This applies to the other cases as well.